.github/workflows/updateGradleLocks.yml

```diff
@@ -0,0 +1,46 @@
+name : Dependabot Gradle Helper
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: 'dependabot/gradle/**'
+
+env:
+  JAVA_VERSION: 11
+
+defaults:
+  run:
+    working-directory: backend
+
+jobs:
+  # in case we once again need to figure out why this thing is not running, some day...
+  # debug-message:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - run: echo "Actor for this run is ${{github.actor}} and event name is ${{github.event_name}}"
+  #       working-directory: /
+  update-locks:
+    # DO NOT run if somebody else (specifically, this workflow) updates a dependabot PR
```

Comment:
The combo of "DO NOT run if" in the explanatory comment and the "run only if" of the actual `if:` boolean is confusing me here. Can this be slightly reworded?

Reply:
```suggestion
    # Run when manually requested or when dependabot creates/updates a PR
    # DO NOT run if somebody else updates a PR (a probably unnecessarily strict guard against infinite 
    # workflow loops, in case we ever enable this workflow to trigger other workflows)
```